primitives/src/block.rs

```diff
@@ -532,4 +594,33 @@ mod tests {
         );
         assert_eq!(format!("{:?}", header), expected);
     }
+
+    #[test]
+    fn header_display() {
+        let header = dummy_header();
+        let expected = "0000000199999999999999999999999999999999999999999999999999999999999999997777777777777777777777777777777777777777777777777777777777777777000000020000000300000004";
```

Comment:
```
let expected = concat!(
    "000000019999999999999999999999999999999999999999999999999999999999999999",
    "7777777777777777777777777777777777777777777777777777777777777777",
   "000000020000000300000004"
);
```

Reply:
Thanks for the idea, check out the new test.

Reply:
nice!